README.md

```diff
@@ -0,0 +1,62 @@
+# buildfarm perf tests
+
+## Purpose
+
+This packages defined some tests which invokes `perf_test` from Apex's [performance_test](https://gitlab.com/ApexAI/performance_test) package. This test allows you to test performance and latency.
+
+* There is a test for each RMW:
+
+  - CycloneDDS (standalone test)
+  - FastRTPS (standalone test)
+  - rmw_connext_cpp.pn
+  - rmw_cyclonedds_cpp
+  - rmw_fastrtps_cpp
+  - rmw_fastrtps_dynamic_cpp
+  - rmw_opensplice_cpp
+
+##  Build
+
+1.  Install ROS 2 (https://index.ros.org/doc/ros2/Installation/).
+1.  Source the ROS 2 installation (either `/opt/ros/<rosdistro>/setup.bash` if installing from binaries, or `ros2_ws/install/setup.bash` if building from source):
+    1.  `source /opt/ros/eloquent/setup.bash`
```

Comment:
For consistency:
```suggestion
    1.  `source /opt/ros/<rosdistro>/setup.bash`
```

Reply:
This regressed. I'm OK with just calling out eloquent if that's easier, but it should be consistent with the reference to the same file above.